src/wallet/db.cpp

```diff
@@ -389,7 +389,11 @@ bool BerkeleyBatch::VerifyEnvironment(const fs::path& file_path, std::string& er
     fs::path walletDir = env->Directory();
 
     LogPrintf("Using BerkeleyDB version %s\n", DbEnv::version(nullptr, nullptr, nullptr));
-    LogPrintf("Using wallet %s\n", walletFile);
+    if (GetWalletDir() == walletDir) {
+        LogPrintf("Using wallet %s\n", walletFile);
+    } else {
+        LogPrintf("Using wallet %s\n", (fs::path(walletDir).filename() / walletFile).string());
```

Comment:
`fs::path(walletDir).filename()` should be written `walletDir.filename()`? :-)

Reply:
@practicalswift right :)

I'm going to implement @promag's suggestion about absolute paths.